src/main/java/base/gameobjects/animalstates/EatingState.java

```diff
@@ -43,6 +41,20 @@ public void update(Animal animal, Game game) {
         }
     }
 
+    private int getSaturationAmount(String foodType) {
```

Comment:
Please add null check here. I know it looks like it can't be null, but due to saving and loading `foodTypeEating` sometimes becomes null